main.go

```diff
@@ -23,6 +23,31 @@ var (
 	skyapiAddr                   string
 )
 
+type requestParams struct {
+	ip            string
+	buildTypeID   string
+	tag           string
+	artifactName  string
+	latestBuildID string
+}
+
+func (r requestParams) log(f llog.LogFunc, msg string, kvs ...llog.KV) {
+	var kv llog.KV
+	if len(kvs) > 0 {
+		kv = kvs[0]
```

Comment:
it basically makes it like an optional parameter. it's the same pattern I used within llog, I figured I'd keep it consistent here